utils/post.js

```diff
@@ -229,8 +229,6 @@ const insertTopics = async (topics = []) => {
           categories: ''
         }
       });
-
-      return topicIndex;
     } catch (e) {
       console.log(e);
       break;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `return topicIndex;` statement might cause issues if the function is expected to return a value. If the `insertTopics` function is used elsewhere in the code and its return value is used, this change could potentially break that functionality. 

If the function doesn't need to return anything, then it's fine. However, if it does, you should add back the return statement.

```diff
+      return topicIndex;
     } catch (e) {
       console.log(e);
       break;
```

Also, using `console.log(e);` to handle errors is not a good practice. It would be better to throw the error or handle it properly so that the caller can decide what to do with it.

```diff
-        console.log(e);
+        throw e;
```
This way, the error can be caught and handled appropriately by the calling function.


<!-- This is an auto-generated comment by OSS CodeRabbit -->